### PR TITLE
Admin webapp

### PR DESCRIPTION
**Pull request by ciphereck**
Depends on #2

### CODE REVIEW

**Comment by ciphereck** [comment]
depends on #2

**Comment by ciphereck** [comment]
#2

File: webapp/src/api/main.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+const API_PREFIX = "https://us-central1-ecommerce-eed77.cloudfunctions.net/api"
+
+export default API_PREFIX;
```
Comment (ankurcharan) [inline webapp/src/api/main.js:3]: use and object here to export such items,
you might need to export another variable and then this'll create or a problem.
or
another possible solution is to create environment variables.

File: webapp/src/components/admin/AdminSubCategory/AdminSubCategory.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    )
+  } else {
+    return (
+      <div>
+        <select value={subCategory} onChange={(event) => setSubCategory(event.target.value)}>
+          {subCategoriesList}
+        </select>
+        <div>
+          <h1>Id:</h1>
+          <TextField id="outlined-basic" label="Outlined" value={subCategories[subCategory].id} variant="outlined" />
```
Comment (ankurcharan) [inline webapp/src/components/admin/AdminSubCategory/AdminSubCategory.js:80]: onchange() event for these?

File: webapp/src/services/util.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import axios from 'axios';
+import {GET_SUB_CATEGORIES, GET_CATEGORIES} from '../api/util';
+import SubCategory from '../model/SubCategory';
+import Category from '../model/Category';
+
+export function getSubCategories() {
+    return axios
+            .get(GET_SUB_CATEGORIES.url)
```
Comment (ankurcharan) [inline webapp/src/services/util.js:8]: why did you create another file for URL and request methods,
there's no use for them there!

**Comment by ciphereck** [comment]
Deleted Forked Repo By Mistake.

**Pull request closed without merging**